platform/metrics/platform-metrics-collector/src/main/java/ddf/metrics/collector/rrd4j/RrdJmxCollector.java

```diff
@@ -268,13 +268,16 @@ void configureCollector() throws CollectorException, IOException {
     LOGGER.trace("ENTERING: configureCollector() for collector {}", mbeanAttributeName);
 
     if (!isMbeanAccessible()) {
-      LOGGER.debug(
-          "MBean attribute {} is not accessible - no collector will be configured for it.",
-          mbeanAttributeName);
-      throw new CollectorException(
-          "MBean attribute "
-              + mbeanAttributeName
-              + " is not accessible - no collector will be configured for it.");
+      String errorMessage = " is not accessible";
+
+      if (Thread.interrupted()) {
```

Comment:
❓ Curious why we added this check. Were you running into problems here?

Reply:
The best way to get the InterruptedException, without making  a lot of changes, seemed to be to piggyback the CollectorException. I wanted it to be more descriptive, so I added to the errorMessage in the case of an interrupt.